CMakeLists.txt

```diff
@@ -55,19 +55,21 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 ################################################################################
 # Extract cluon-msc from cluon-complete.hpp.
+
+execute_process(COMMAND wget -P ${CLUON_PATH} https://raw.githubusercontent.com/chrberger/libcluon/gh-pages/headeronly/${CLUON_COMPLETE})
```

Comment:
@arbulgazar Is this line executed once or on any build?

Reply:
@chrberger yes, executed on every build, but if the file is already there it is not downloaded again. Should I make this check explicitly?

Reply:
The issue might be that if you want to build when you only have the repository at hand and no Internet connection, this step might fail?

Reply:
Regardless - will add an explicit check, will be cleaner.

Reply:
If you run this outside of the docker, e.g. natively, it will clutter the repository.